apps/volontulo/views/organizations.py

```diff
@@ -137,7 +137,7 @@ def organization_form(request, slug, id_):  # pylint: disable=unused-argument
         else:
             messages.error(
                 request,
-                u"Należy wypełnić wszystkie pola formularza."
+                "Należy wypełnić wszystkie pola formularza."
```

Comment:
it would be nice to if we have coherent quotation marks in a file (sometimes we have single quotes sometimes double)